orgs/fram.json

```diff
@@ -23,8 +23,8 @@
     },
 
     "supportUrl": {
-      "default": "https://frammr.no/kontakt-oss",
-      "en-US": "https://en.frammr.no/contact-us"
+      "default": "https://reise.frammr.no/contact",
```

Comment:
Will this be the main entry to the new contact form? Is FRAM removing the other contact page (https://frammr.no/kontakt-oss)?

Reply:
Yes, this will be the main entry of the new contact form. FRAM will eventually remove the link to the old contact page.